src/functions/module_lookup.ts

```diff
@@ -661,22 +681,23 @@ export const XLOOKUP = {
     arg("if_not_found (any, optional)", _t("If a valid match is not found, return this value.")),
     arg(
       `match_mode (any, default=${DEFAULT_MATCH_MODE})`,
-      _t(
-        "(0) Exact match. \
-        (-1) Return next smaller item if no match. \
-        (1) Return next greater item if no match. \
-        (2) Wildcard match."
-      )
+      _t("Specifies how to match search_key with the items in lookup_range. "),
+      [
+        { value: 0, label: _t("Exact match (default)") },
+        { value: -1, label: _t("Exact match or next smaller item") },
+        { value: 1, label: _t("Exact match or next larger item") },
+        { value: 2, label: _t("Wildcard character match") },
+      ]
     ),
     arg(
       `search_mode (any, default=${DEFAULT_SEARCH_MODE})`,
-      _t(
-        "(1) Search starting at first item. \
-      (-1) Search starting at last item. \
-      (2) Perform a binary search that relies on lookup_array being sorted in ascending order. If not sorted, invalid results will be returned. \
-      (-2) Perform a binary search that relies on lookup_array being sorted in descending order. If not sorted, invalid results will be returned.\
-      "
-      )
+      _t("Specifies the search mode to use. By default, a first to last search will be used."),
+      [
+        { value: 1, label: _t("Search first to last (default)") },
+        { value: -1, label: _t("Search last to first") },
+        { value: 2, label: _t("Binary search, (sorted ascending order)") },
+        { value: -2, label: _t("Binary search, (sorted descending order)") },
+      ]
```

Comment:
the comma looks weird with the parenthesis
```suggestion
        { value: 2, label: _t("Binary search (sorted ascending order)") },
        { value: -2, label: _t("Binary search (sorted descending order)") },
      ]
```